src/plugins/intel_npu/src/plugin/npuw/partitioning/partitioning.cpp

```diff
@@ -2068,7 +2194,6 @@ ov::npuw::Partitioning ov::npuw::getPartitioning(const std::shared_ptr<ov::Model
                 p.saveTinyConstants(func_group);
                 p.saveScaleFactors(func_group);
                 p.createFunction(func_group);
-                p.optimize(func_group);
```

Comment:
Why is it gone for CWAI?

Reply:
since the whole idea behind CWAI is to keep dequant params in the model, and the whole idea of `optimize` is to work on partitioned/folded subgraphs.

There are changes that may serve both (e.g., host-side regather, PMM, etc) so maybe it makes to bring `optimize` back to CWAI or just separate it somehow. Actually, DQ shouldn't work on on the CWAI-d model - patterns won't match.